### DISCUSSION TITLE
I'm a US citizen...

### DISCUSSION THREAD

**Discussion by s4w3d0ff** [s4w3d0ff/python-poloniex#208] [General]
Since Poloniex is no longer accepting new accounts or allowing trades for US based customers, I can no longer test most of the private api commands. 

I'm not sure how this will effect this repository in the future. For now I will do my best to keep maintaining this repository, but it is hard to confidently push out new code that isn't tested on the more important side of the api (private commands).

**Comment by teknostatik7** [comment]
Understood, I'm in the same boat. Unfortunately our only option now is Coinbase.

**Comment by pinarmeltem** [comment]
I have a Poloniex account and I'm not a US citizen. I can test, I guess.
